tests/handlers/protocols-query.spec.ts

```diff
@@ -117,23 +120,45 @@ export function testProtocolsQueryHandler(): void {
         });
 
         const conditionalQueryReply = await dwn.processMessage(alice.did, conditionalQuery.message);
-
         expect(conditionalQueryReply.status.code).to.equal(200);
         expect(conditionalQueryReply.entries?.length).to.equal(1); // only 1 entry should match the query on protocol
 
         const protocolConfigured = conditionalQueryReply.entries![0] as ProtocolsConfigureMessage;
         expect(protocolConfigured).to.deep.equal(protocol2.message);
 
-        // testing fetch-all query without filter
+        // testing authenticated but unauthorized (to private ProtocolsConfigures) conditional query
+        const signedConditionalQuery = await ProtocolsQuery.create({
+          filter : { protocol: protocol2.message.descriptor.definition.protocol },
+          signer : Jws.createSigner(bob)
+        });
+
+        const signedConditionalQueryReply = await dwn.processMessage(alice.did, signedConditionalQuery.message);
+        expect(signedConditionalQueryReply.status.code).to.equal(200);
+        expect(signedConditionalQueryReply.entries?.length).to.equal(1); // only 1 entry should match the query on protocol
+
+        const protocolConfigured2 = conditionalQueryReply.entries![0] as ProtocolsConfigureMessage;
+        expect(protocolConfigured2).to.deep.equal(protocol2.message);
+
+        // testing unauthenticated fetch-all query without filter
         const fetchAllQuery = await ProtocolsQuery.create({
         });
 
         const fetchAllQueryReply = await dwn.processMessage(alice.did, fetchAllQuery.message);
-
         expect(fetchAllQueryReply.status.code).to.equal(200);
         expect(fetchAllQueryReply.entries?.length).to.equal(2);
         expect(fetchAllQueryReply.entries).to.deep.include(protocol2.message);
         expect(fetchAllQueryReply.entries).to.deep.include(protocol3.message);
+
+        // testing authenticated but authorized (to private ProtocolsConfigures) fetch-all query without filter
```

Comment:
Same comment as above, but also  thing you meant unauthorized here.
```suggestion
        // testing authenticated but unauthorized (to private ProtocolsConfigures) fetch-all query without filter
```

Reply:
Fixed!